app/styles/base.css

```diff
@@ -654,6 +654,79 @@ select:active {
 }
 
 /* ----------------------------------------
+ * Loading Animation
+ * ----------------------------------------
+ */
+
+#noVNC_loading_anim {
```

Comment:
Ah, did not realize we had an existing `noVNC_loading` class.  I'll tweak things a bit.